source/_includes/asides/component_navigation.html

```diff
@@ -28,6 +28,19 @@
     </div>
   {% endif %}
 
+  {% if page.ha_qa_scale %}
+    <div class='section'>
+      Quality Scale: 
+        {% if page.ha_qa_scale == 'platinum' %}
+          <a href='https://developers.home-assistant.io/docs/en/next/integration_quality_scale_index.html#platinum'>🏆</a>
```

Comment:
I think that we should add a page about the Integration Quality Scale to the normal documentation and make that user focused. It should not include the requirements but just the descriptions of the different levels.

Reply:
That's a valid point.

Reply:
We can just start with a page in the docs that just mentions the existing descriptions from the dev site.

Reply:
This link still needs updating.